wayne927-toronto.md

```diff
@@ -0,0 +1,7 @@
+## Toronto ON
+
+### Great places to eat
```

Comment:
@wayne927 Boston isn't all that far, but we haven't been to Canada for several years so we can't make use of great places to eat :-(